test/gtest/ucp/test_ucp_wfe.cc

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+*
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
```

Comment:
not needed?

Reply:
Since it was in the original perf file that I copied for this file, I have to preserve the copyright (BSD3 requirements)

Reply:
> Since it was in the original perf file that I copied for this file, I have to preserve the copyright (BSD3 requirements)

👍